README.md

```diff
@@ -28,32 +31,80 @@ The decision to fork the original mender-mcu-client was made to:
 * **Provide Official Support**: Ensure that the project receives the necessary attention and
   resources from Northern.tech to meet the needs of the community and enterprise users.
 
+## Getting started
 
-## Get Started
-
-Since the project is under active development, we recommend watching the repository or checking back
-regularly for updates. Detailed documentation and usage instructions will be provided as the project
-progresses.
-
+For using `mender-mcu` for the first time, we strongly recommend you start with our [Zephyr reference application](https://github.com/mendersoftware/mender-mcu-integration).
 
-## Get started with Zephyr OS
-
-For using `mender-mcu` for the first time, we strongly recommend you start with our Zephyr reference
-application.
-
-It can be found in the repository
-[mender-mcu-integration](https://github.com/mendersoftware/mender-mcu-integration). The policy is
-that the reference application's `main` branch will follow the latest stable branch of `mender-mcu`
+The policy is that the reference application's `main` branch will follow the latest stable branch of `mender-mcu`
 (this repository).
 
-In the `mender-mcu-repository` you will find:
+In the `mender-mcu-integration` repository you will find:
 * A reference `west` workspace for Zephyr OS builds, including `CMake` and `KConfig` configurations.
 * Sample application for how to use `mender-mcu` APIs.
 * A list of boards that we have so far tested.
 
 We recommend starting there to understand how we have designed our solution and then coming back
 here for more in-depth information.
 
+### Compatibility
+| Zephyr OS version | Bootloader |
+|-------------------|------------|
+| v3.7.0            | MCUboot    |
+
+### Boards
+The reference board for `mender-mcu` is the [ESP32-S3-DevKitC](https://docs.espressif.com/projects/esp-dev-kits/en/latest/esp32s3/esp32-s3-devkitc-1/index.html#).
+
+To build the project, you need a board that supports MCUboot. The following link will filter
+the officially supported boards that also support MCUboot:
+
+* [Zephyr Project supported boards with MCU boot](https://docs.zephyrproject.org/latest/gsearch.html?q=MCUboot&check_keywords=yes&area=default#gsc.tab=0&gsc.q=MCUboot&gsc.ref=more%3Aboards&gsc.sort=)
+
```

Comment:
This is not true. Only if you want to use our module. Otherwise you can develop your own module using the provided API.

So this I would move to the section where we introduce Update Modules.